apps/keyboard/js/views/candidate_panel_view.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
+
+/* global BaseView */
+
+(function(exports) {
+
+/**
+ * CandidateView handles the rendering of each candidate, i.e.
+ * the word suggestion.
+ */
+function CandidateView() {
+  BaseView.apply(this, arguments);
+}
+
+CandidateView.prototype = Object.create(BaseView.prototype);
+
+CandidateView.prototype.render = function render() {
+  var span = document.createElement('span');
+  span.textContent = this.target.text;
+  span.style.flex = this.options.unit;
+  // ui/integration test needs this
+  span.dataset.data = this.target.data;
+
+  this.element = span;
+
+  if (this.viewManager) {
```

Comment:
I prefer this way, so that we don't need to pass a viewManager into this View in some cases, like in tests.
Please let me know if you think this is not a good practice.